pkg/test/framework/components/istio/operator.go

```diff
@@ -243,11 +290,56 @@ func deployControlPlane(c *operatorComponent, cfg Config, cluster kube.Cluster,
 		"--wait",
 	}
 	cmd = append(cmd, installSettings...)
-	scopes.CI.Infof("Running istio control plane on cluster %s %v", cluster.Name(), cmd)
-	if _, _, err := istioCtl.Invoke(cmd); err != nil {
-		return fmt.Errorf("manifest apply failed: %v", err)
+	errG.Go(func() error {
```

Comment:
Above, in the `!controlPlane` case, another goroutine runs to constantly try replacing the istiod-remote stuff so that `apply` doesn't hang waiting for IngressGateway to ready. It's pretty hacky.